packages/storefront-lwc/src/modules/commerce/errorPage/errorPage.scss

```diff
@@ -0,0 +1,28 @@
+pre {
+    background: #303030;
+    color: #f1f1f1;
+    padding: 10px 16px;
```

Comment:
For consistency, please em instead of px.

Reply:
Done.